plugins/kubernetes/src/api/types.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createApiRef } from '@backstage/core';
+
+export const kubernetesApiRef = createApiRef<KubernetesBackendApi>({
+  id: 'plugin.kubernetes.service',
+  description:
+    'Used by the Kubernetes plugin to make requests to accompanying backend',
```

Comment:
Same here, it's more about exposing k8s objects rather than talking to the backend.